utils/cadence/args.go

```diff
@@ -0,0 +1,20 @@
+package cadence
```

Comment:
Wouldn't it be better to put this to some local utils in `access` package?

Reply:
idk. I'd put it in `engine/access/utils/` but the validator is in `access/`, so there will be a circular dependency. maybe `access/utils` is the option?
I'd put it locally until it is used not only in our package.

Reply:
`access/utils` works